src/components/chat/comment-card.tsx

```diff
@@ -93,7 +93,7 @@ export const CommentCard: React.FC<IProps> = ({ activeNavTab, user, postedCommen
                 {
                   appConfig.showCommentTag && comment.tags &&
                   <div className="comment-dropdown-tag">
-                    { comment.tags.join(", ") }
+                    { Object.values(comment.tags).join() || comment.tags[0] }
```

Comment:
I added a fallback just in case. 
As for the original `comments.tags.join()`, this kept crashing even when I created the field inside Firestore (see below) 
![image](https://github.com/concord-consortium/collaborative-learning/assets/521934/4540c91e-bc14-4bd6-8368-924f8d537f02)

I even console logged it, saw it was an array. Saw its `typeof`  - which was `object` - expected in JS.  and `Array.isArray(comment.tags) `returned truthy - but when I went to actually render it
` { comment.tags.join(", ") }
`
OR 
` { comment.tags.join() }`

It crashed the program.

Reply:
Also note that `Object.values(comment.tags).join() ` should work only when the `firestore-schema.ts` has updated to include `tags?: string[]` (see picture below )
<img width="1010" alt="image" src="https://github.com/concord-consortium/collaborative-learning/assets/521934/c3223e88-e96e-4d44-a8a5-a42976ebf7b5">

That change was made in an earlier PR.
Can we verify if this was deployed?